assets/javascripts/modules/autoComplete.js

```diff
@@ -136,6 +138,7 @@ var updateInput = function (title, value) {
 var updateTargetInput = function (value) {
   if ($targetInput) {
     $targetInput.val(value);
+    $targetInput.change();
```

Comment:
not necessary for this, just a habit, because setting a value on a select does not fire the native 'change' event
see https://api.jquery.com/val/#entry-longdesc-1